README.md

```diff
@@ -8,6 +8,7 @@
 
 [![Build Status](https://travis-ci.org/mailjet/mailjet-apiv3-go.svg?branch=master)](https://travis-ci.org/mailjet/mailjet-apiv3-go)
 [![GoDoc](https://godoc.org/github.com/mailjet/mailjet-apiv3-go?status.svg)](https://godoc.org/github.com/mailjet/mailjet-apiv3-go)
+[![Go Report Card](https://goreportcard.com/badge/mailjet/mailjet-apiv3-go)](https://goreportcard.com/report/mailjet/mailjet-apiv3-go)
```

Comment:
I checked both url and they work
https://goreportcard.com/badge/mailjet/mailjet-apiv3-go
https://goreportcard.com/report/mailjet/mailjet-apiv3-go